src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,86 @@
+    ```
+
+1. Send the chain’s gateway and prover addresses to the Interop Labs team by filling out the [Google form](TODO: ADD LINK). The team will register the chain and gateway with the router and authorize the prover.
+
+## Begin routing messages
+
+Once your chain is registered with the protocol, it will need verifiers to begin routing messages. See [Become an Amplifier Verifier (Worker)](/validator/amplifier/verifier-onboarding) for instructions on onboarding a verifier.
```

Comment:
Should clarify that the onboarding is happening on devnet-amplifier and not devnet-verifier for chain integrators doing it (separate from the onboarding process we're doing with axelar validators)